bpaotu/bpaotu/settings.py

```diff
@@ -362,3 +362,6 @@
 EXPLORER_TASKS_ENABLED = True
 EXPLORER_TOKEN_AUTH_ENABLED = True
 EXPLORER_TOKEN = env.get('EXPLORER_TOKEN', "EXPLOREIT")
+
+# to make ckan authentication optional
```

Comment:
maybe say `# enable integration with CKAN authentication (specific to the BPA data portal)`

Reply:
Fixed